app/lib/actions/katello/content_view_puppet_module/destroy.rb

```diff
@@ -29,9 +30,8 @@ def plan(repository)
             if content_view_puppet_modules.any?
               puppet_repoids = ::Katello::Repository.puppet_type.in_environment(repository.organization.library).
                   pluck(:pulp_id).reject { |repoid| repoid == repository.pulp_id }
-              found_puppet_module = ::Katello::PuppetModule.latest_modules_search([{:name => puppet_module.name,
-                                                                                    :author => puppet_module.author}],
-                                                                                  puppet_repoids).first
+              found_puppet_module = ::Katello::PuppetModule
+                .latest_module(puppet_module.name, puppet_module.author, puppet_repoids)
```

Comment:
as @daviddavis said a bunch of us do prefer 100 character limits (rubocop isn't even enforcing less than 200 it appears likely due to legacy code)

Reasons for this include 
* modern editors and monitors
* increased readability

I tend to prefer the way the code was before as its more readable, but don't feel strongly either way.  I would not expect all commits to conform to 80 chars though.